src/pages/lists/action-list/action-list-local-actions/device-edit.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { createStyles, makeStyles } from '@material-ui/core';
+import Divider from '@material-ui/core/Divider';
+
+type DialogProps = {
+    open: boolean;
+    handleClose: () => void;
+    subTitle: string;
+    updateSubTitle: (tempSubTitle: string) => void;
+};
+
+const useStyles = makeStyles(() =>
+    createStyles({
+        dialogPaper: {
+            width: 450,
+            height: 600,
+        },
+        dialogActions: {
+            flexDirection: 'column',
+            padding: 0,
+        },
+        dialogButton: {
+            margin: '16px',
+            width: 'calc(100% - 32px)',
+        },
```

Comment:
For dialog boxes, on desktop, we do 24px spacing. So all the RED arrows should indicate a 24 px spacing. You did very well on the top; but around the "done" button it is off.

![image](https://user-images.githubusercontent.com/8997218/145492557-edc44697-2a72-490d-9839-2493b89481dc.png)